fsharp-backend/src/BackendOnlyStdLib/HttpClient.fs

```diff
@@ -130,21 +137,25 @@ let makeHttpCall
         // dynamically upgraded)
         req.Version <- System.Net.HttpVersion.Version20
 
-        // username and password - note that an actual auth header will overwrite this
+        // If present, extracts username/password from URI and passes along as Basic Auth
+        //
+        // e.g. expecting form http://user:password@www.contoso.com/index.htm
+        // Note that an actual auth header will overwrite this
         if uri.UserInfo <> "" then
           let authString =
             // UserInfo is escaped during parsing, but shouldn't actually isn't
             // useful here, so unescape it.
             let userInfo = System.Uri.UnescapeDataString uri.UserInfo
             // Handle usernames with no colon
             if userInfo.Contains(":") then userInfo else userInfo + ":"
+
           req.Headers.Authorization <-
             Headers.AuthenticationHeaderValue(
               "Basic",
               System.Convert.ToBase64String(UTF8.toBytes authString)
             )
 
-        // content
+        // Write content to request body if relevant
```

Comment:
Im not super sure these comments add context.

Their original intent was not to describe what was happening, but to delineate sections. So the sections were "username and password", "content", "headers", "send request".

I think these new comments redescribe what's in the code without providing new info, and they make it less clear at a glance of what the sections are.

Reply:
that's fair - I can revert those